bluebottle/funding/transitions.py

```diff
@@ -25,12 +26,32 @@ def target_reached(self):
         if not self.instance.amount_raised >= self.instance.target:
             return _("Amount raised should at least equal to target amount.")
 
+    def is_complete(self):
+        errors = [
+            _('{} is required').format(self.instance._meta.get_field(field).verbose_name)
+            for field in self.instance.required
+        ]
+
+        if errors:
+            return errors
+
+    def is_valid(self):
+        errors = [
+            error.message[0] for error in self.instance.errors
+        ]
+
+        if errors:
+            return errors
+
```

Comment:
Why do we not use the methods on the parent class?

Reply:
Because `self` will then expect a `ActitivityTransitions` instance, but a `FundingTransitions` will be provided. I guess there is a nicer way to do this, but I couldn't work it out.